PS4k/src/com/vitco/manager/action/types/SelectLogicAction.java

```diff
@@ -0,0 +1,49 @@
+package com.vitco.manager.action.types;
```

Comment:
I don't think you need to put it into a separate file. Have a look how I did it with the Camera:
https://github.com/simlu/voxelshop/blob/master/PS4k/src/com/vitco/layout/content/mainview/MainView.java#L388